### PR TITLE
Change All Bigint Ref

### PR DESCRIPTION
**Pull request by rpsoftech**
While Compiling It with some old library like @ziet/ncc

(node:9454) UnhandledPromiseRejectionWarning: Error: Module parse failed: Identifier directly after number (669:86)

### CODE REVIEW

**Comment by rusher** [comment]
This is normally not an issue, because node.js support bigint for all supported version (and if not supported, an error message is thrown).
Could you indicate the node.js version you are using ?

**Comment by rpsoftech** [comment]
it was not thrown by nodejs.

It was by webpack while compiling nodejs code to singal file.

File: test/integration/datatype/test-integer.js [hunk 1]
```diff
@@ -121,20 +121,20 @@ describe('integer with big value', () => {
...
-        assert.strictEqual(rows[1].v, 127n);
-        assert.strictEqual(rows[2].v, 128n);
-        assert.strictEqual(rows[3].v, 9007199254740991n);
-        assert.strictEqual(rows[4].v, 9007199254740992n);
-        assert.strictEqual(rows[5].v, 9007199254740993n);
+        assert.strictEqual(rows[0].v, BigInt(-9007199254740991));
+        assert.strictEqual(rows[1].v, BigInt(127));
+        assert.strictEqual(rows[2].v, BigInt(128));
+        assert.strictEqual(rows[3].v, BigInt(9007199254740991));
+        assert.strictEqual(rows[4].v, BigInt(9007199254740992));
```
Comment (rusher) [inline test/integration/datatype/test-integer.js:128]: BigInt(9007199254740992) is unpredicable  that has to be replaced by string -> BigInt('9007199254740992')

File: test/integration/datatype/test-integer.js [hunk 2]
```diff
@@ -121,20 +121,20 @@ describe('integer with big value', () => {
...
+        assert.strictEqual(rows[5].v, BigInt(9007199254740993));
```
Comment (rusher) [inline test/integration/datatype/test-integer.js:129]: same here BigInt(9007199254740993) => BigInt('9007199254740993')

File: test/integration/datatype/test-integer.js [hunk 3]
```diff
@@ -121,20 +121,20 @@ describe('integer with big value', () => {
...
         assert.strictEqual(typeof rows[4].v, 'bigint');
         return base.createConnection({ supportBigInt: true });
       })
       .then((conn2) => {
         conn2
           .query('INSERT INTO testBigint values ()')
           .then((rows) => {
-            assert.strictEqual(rows.insertId, 9007199254740994n);
+            assert.strictEqual(rows.insertId, BigInt(9007199254740994));
```
Comment (rusher) [inline test/integration/datatype/test-integer.js:137]: BigInt('9007199254740994')

File: test/integration/datatype/test-integer.js [hunk 4]
```diff
@@ -252,7 +252,7 @@ describe('integer with big value', () => {
   });
 
   it('using very big number bigint', function (done) {
-    const maxValue = 18446744073709551615n;
+    const maxValue = BigInt(18446744073709551615);
```
Comment (rusher) [inline test/integration/datatype/test-integer.js:255]: BigInt('18446744073709551615')

**Review by rusher — CHANGES_REQUESTED** [review]
Ok, i didn't understand initial concern: Bigint literal are not supported by webpack. 
So this changed can make sense. Tests are just to be corrected accordingly

**Comment by rusher** [comment]
Thanks for correcting test. 
This is now merged in develop branch, and will be release in next 2.5.3 version

**Pull request closed without merging**